### PR TITLE
Improve login accuracy event on Laravel>5

### PR DESCRIPTION
**Pull request by estringana**
### Description

It was reported that success login event on Laravel was being reported on authenticated user instead on the login event. This PR fixes that.

### Reviewer checklist
- [ ] Test coverage seems ok.
- [ ] Appropriate labels assigned.

### CODE REVIEW

File: tests/Common/WebFrameworkTestCase.php [hunk 1]
```diff
@@ -21,12 +21,36 @@ abstract class WebFrameworkTestCase extends IntegrationTestCase
...
     protected $checkWebserverErrors = true;
+    protected $cookiesFile;
+    protected $maintainSession = false;
+
+    protected function ddSetUp()
+    {
+        parent::ddSetUp();
+    }
+
+    protected function enableSession()
```
Comment (estringana) [inline tests/Common/WebFrameworkTestCase.php:39]: This PRs introduces integration tests in which a session need to be maintained in multiple consecutive calls. That's the reason for this

File: src/DDTrace/Integrations/Laravel/LaravelIntegration.php [hunk 2]
```diff
@@ -350,7 +350,7 @@ function ($This, $scope, $args, $loginSuccess) use ($integration) {
 
         // Used by Laravel >= 5.0
         \DDTrace\hook_method(
-            'Illuminate\Auth\Events\Authenticated',
```
Comment (estringana) [inline src/DDTrace/Integrations/Laravel/LaravelIntegration.php:353]: This is the only core change. Everything else is test code to ensure the fix is working

**Review by Anilm3 — APPROVED** [review]
LGTM, although double checking on the weblog before merging would be great.

**Review by PROFeNoM — APPROVED** [review]
👌

**Pull request merged**